source/services/index.md

```diff
@@ -0,0 +1,105 @@
+An `Ember.Service` is a long-lived `Ember.Object` available application-wide. Services are singletons, and only one instance of a given service can exist at a time.
```

Comment:
the "and" reads weird, since you're giving the definition of what a singleton is.